ipatests/test_webui/test_hostgroup.py

```diff
@@ -31,6 +31,15 @@
 import pytest
 
 
+def test_invalid_names(self, names, error_link):
+    for name in names:
+        self.navigate_to_entity(hostgroup.ENTITY)
```

Comment:
the `navigate` line could be above the for loop, as only one navigation should happen, right?